js/src/features/human-resouce/epics.ts

```diff
@@ -238,15 +241,13 @@ export const specifySkillsEpic: RootEpic = (action$, state$, { api }) =>
 export const addEmployeeNoteEpic: RootEpic = (action$, state$, { api }) =>
   action$.pipe(
     filter(isActionOf(addEmployeeNoteAsync.request)),
-    debounceTime(500),
     switchMap((action) =>
       from(api.humanResource.addNote(action.payload)).pipe(
-        map((result) =>
+        mergeMap((result) =>
           result.success
-            ? addEmployeeNoteAsync.success()
-            : addEmployeeNoteAsync.failure(result.reason),
+            ? of(addEmployeeNoteAsync.success(), closeCreateNoteModal())
```

Comment:
If you need to return multiple actions, use [addEmployeeNoteAsync.success (), closeCreateNoteModal ()], because mergeMap returns an Observable

Reply:
done